ci/blackbox/rdsbroker_test.go

```diff
@@ -0,0 +1,228 @@
+package integration_aws_test
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+
+	uuid "github.com/satori/go.uuid"
+
+	. "github.com/alphagov/paas-rds-broker/ci/helpers"
+)
+
+const (
+	INSTANCE_CREATE_TIMEOUT = 30 * time.Minute
+)
+
+var _ = Describe("RDS Broker Daemon", func() {
+	BeforeEach(func() {
+	})
+
+	AfterEach(func() {
+	})
+
+	It("should check the instance credentials", func() {
+		Eventually(rdsBrokerSession, 30*time.Second).Should(gbytes.Say("credentials check has ended"))
+	})
```

Comment:
Shall we add more tests around this? Lets say: We create a instance in the suite and we check that the password is updated... although, tbh, it is a PITA, we need to wait for that instance to get created and so on.